app/components/pan-zoom.spec.js

```diff
@@ -39,11 +39,11 @@ describe('PanZoom', function () {
       let wrapper;
       let zoomInButton;
 
-      beforeEach(function () {
+      before(function () {
```

Comment:
I think these tests were broken because `beforeEach` attached a new copy of the component to the document body for each test.

I'm not really sure how that worked before.